erpnext/public/js/utils.js

```diff
@@ -889,6 +889,11 @@ erpnext.utils.map_current_doc = function(opts) {
 	}
 }
 
+erpnext.utils.has_valuation_read_permission = function() {
+	let allowed_role = frappe.defaults.get_default('restrict_amounts_in_report_to_role');
+	return frappe.user.has_role(allowed_role);
```

Comment:
Should match the same condition as in backend
`return !allowed_role || frappe.user.has_role(allowed_role);`

Reply:
yup :P